package.json

```diff
@@ -17,49 +17,11 @@
         "pace": "0.0.4",
         "resolve-url-loader": "^4.0.0",
         "sass": "^1.55.0",
-        "sass-loader": "^9.0.3",
-        "vue-template-compiler": "^2.7.8"
+        "sass-loader": "^9.0.3"
     },
     "dependencies": {
         "@coreui/coreui": "^2.1.16",
         "@digitallyhappy/backstrap": "^0.4.0",
-        "animate.css": "^4.1.1",
-        "bootstrap": "^4.6.0",
-        "bootstrap-colorpicker": "^3.4.0",
-        "bootstrap-datepicker": "^1.9.0",
-        "bootstrap-daterangepicker": "^3.1.0",
-        "bootstrap-iconpicker": "^1.8.2",
-        "ckeditor4": "^4.20.0",
-        "cropperjs": "^1.5.12",
-        "datatables.net": "^1.12.1",
-        "datatables.net-bs4": "^1.12.1",
-        "datatables.net-fixedheader": "^3.2.4",
-        "datatables.net-fixedheader-bs4": "^3.2.4",
-        "datatables.net-responsive": "^2.3.0",
-        "datatables.net-responsive-bs4": "^2.3.0",
-        "easymde": "^2.18.0",
-        "jquery": "^3.6.0",
-        "jquery-colorbox": "^1.6.4",
-        "jquery-cropper": "^1.0.1",
-        "jquery-ui-dist": "^1.13.1",
-        "jquery-ui-touch-punch": "^0.2.3",
-        "line-awesome": "1.3.0",
-        "moment": "^2.29.4",
-        "nestedSortable": "^1.3.4",
-        "nonblockjs": "^1.0.8",
-        "noty": "^3.2.0-beta",
-        "pace-js": "^1.2.4",
-        "pc-bootstrap4-datetimepicker": "^4.17.51",
-        "perfect-scrollbar": "^1.5.5",
-        "places.js": "^1.19.0",
-        "popper.js": "^1.16.1",
-        "select2": "^4.0.13",
-        "select2-bootstrap-theme": "0.1.0-beta.10",
-        "simple-line-icons": "^2.5.5",
-        "source-sans-pro": "^3.6",
-        "summernote": "^0.8.20",
-        "sweetalert": "^2.1.2",
-        "tinymce": "^5.10.2",
-        "urijs": "^1.19.11"
```

Comment:
I haven't commited the `packages` directory itself, it would make the number of changed files EXPLODE. But look at how clean the `packages` directory is now, after you run `npm install && npm run prod` in CRUD:

![CleanShot 2023-01-03 at 23 34 18](https://user-images.githubusercontent.com/1032474/210445052-bab28e61-173d-4fdf-90b3-632dd46f61fc.png)

I've chosen to leave these 3, because:
- [ ] we're still bundling `backstrap` and the custom styles here together; but we should move these custom styles to `backstrap`, bundle it there, so we can then include it from CDN directly;
- [ ] we still haven't solved loading fonts from Google CDN or such; so right now we're loading them from local; but once we solve that, we should be able to remove the font from here too;

Reply:
After we fix the two items above, we should be able to remove:
- `package.json` file
- `webpack.mix.js` file
- the `public/packages` directory entirely

That's when we have our first theme, the `base` theme.

Reply:
Done! Removed all of those files 🎉